tasks/filesystem-sudoers.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Install sudo
+  apt:
+    name: sudo
+    update_cache: yes
+  when: ansible_distribution == "Debian" or ansible_distribution == "Kali GNU/Linux"
+  become: yes
```

Comment:
This was already merged in a0cdb4c029a528557c2c488f3a0c93c9e1846ae1 (see 83bf911f5327827468497fca42fe110a4cbe4f3b).